docs/Remote Control Guide/index.md

```diff
@@ -0,0 +1,184 @@
+
+## Remote Control Modules
+
+### Climate
+
+The climate module consists of climate sub-modules represented by a `ClimateControlCapabilities` object. Each sub-module exposes many aspects of a car's climate controls, such as setting the desired temperature or turning on the heated windshield. 
+
+### Radio
+
+The radio module consists of radio sub-modules represented by a `RadioControlCapabilities` object. Each sub-module exposes many aspects of a car's radio controls, such as setting the desired frequency and band the radio is operating on.
+
+### Seat
+
+The seat module consists of seat sub-modules repesented by a `SeatControlCapabilities` object. Each sub-module exposes many aspects of a car's seat controls, such as setting the back tilt angle and the massage mode.
+
+### Audio
+
+The audio module consists of audio sub-modules represented by a `AudioControlCapabilities` object. Each sub-module exposes many aspects of a car's audio controls, such as setting the volume or modifying the equalizer settings.
+
+### Light
+
+The light module does not contain any sub-modules but instead has an array of `LightCapabilities` objects, each identified by a `LightName`. This module exposes the ability to modify attributes such as the brightness and color of each light.
+
+### HMI Settings
+
+The HMI settings module does not contain any sub-modules and is represented by an `HMISettingsControlCapabilities` object. This module exposes the the ability to set the desired temperature and distance units as well as toggle the display mode of the HMI between night and day.
+
+### Button
+
+Button is an interesting remote control component because it is not a remote control module. `RemoteControlCapabilities` includes an array of `ButtonCapabilities` structs which describe either a physical button or a softbutton. A mobile app may send a `ButtonPress` RPC with the `ButtonName` and `moduleId` from any of these `ButtonCapabilities` to perform an action on another remote control module.
+
+## Consent
+
+The behavior of module allocation in SDL core is shown in the following table:
+
+!!! NOTE
+The driver is always considered to be within the service area.
+SDL will assume actions performed by the driver are consented to by the driver.
+!!!
+
+|User Location|Allow Multiple Access|Requesting App HMI Level|Requested Module State|Access Mode|SDL Action|
+|:-----|:---|:---|:---|:---|:---|
+|out of service area|any|any|any|any|disallow|
+|in service area|true|any|any|any|allow|
+|in service area|false|any|free|any|allow|
+|in service area|false|any|busy|any|disallow|
+|in service area|false|any|in use|any|disallow|
+|in service area|true|BACKGROUND or LIMITED|in use|any|disallow|
+|in service area|true|FULL|in use|auto allow|allow|
+|in service area|true|FULL|in use|auto deny|disallow|
+|in service area|true|FULL|in use|ask driver|ask driver|
+
+#### Requested Module State
+
+"free" indicates no application currently holds the requested resource
```

Comment:
Might be better to put these in separate bullet points

Reply:
added in https://github.com/smartdevicelink/sdl_core_guides/pull/61/commits/1ef97a863cfbfb865722057695a75b2e7f8f2407